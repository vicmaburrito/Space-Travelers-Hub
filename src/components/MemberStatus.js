import React from 'react';
import PropTypes from 'prop-types';

function MemberStatus({ reserved }) {
  if (reserved === false) return <badge className="p-1 bg-secondary text-white rounded">NOT A MEMBER</badge>;
  return <badge className="p-1 bg-info text-white rounded">Active Member</badge>;
}

export default MemberStatus;

MemberStatus.propTypes = {
  reserved: PropTypes.bool,
};

MemberStatus.defaultProps = {
  reserved: false,
};
