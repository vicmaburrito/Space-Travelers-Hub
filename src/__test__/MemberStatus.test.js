import React from 'react';
import renderer from 'react-test-renderer';
import MemberStatus from '../components/MemberStatus';

it('renders correctly', () => {
  const tree = renderer.create(<MemberStatus />).toJSON();
  expect(tree).toMatchSnapshot();
});
