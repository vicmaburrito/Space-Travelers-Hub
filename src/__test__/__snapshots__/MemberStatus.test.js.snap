// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="p-1 bg-secondary text-white rounded"
>
  NOT A MEMBER
</span>
`;
